### PR TITLE
BR-62: Add eslint and husky

### PR DESCRIPTION
**Pull request by jeissonmoncada001**
## Jira link
[BR-62](https://mundobakia.atlassian.net/browse/BR-62)

## Description
Add eslint and husky

## How do I test this PR?
- Clone the repository
- Install the dependencies using the 'yarn' command
- Run the application locally using 'yarn start' command

## Screenshots
Captura de Pantalla 2022-04-05 a la(s) 4 43 31 p m
Captura de Pantalla 2022-04-05 a la(s) 4 38 31 p m

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
```
Comment (Julian9411) [inline .eslintrc.json:1]: Why this file is a JSON? 
Why this file is not similar or is not same to eslint.js in the timing project

Comment (jeissonmoncada001) [reply .eslintrc.json]: Done!

**Comment by Julian9411** [comment]
Where is pre-comit file?

**Comment by Julian9411** [comment]
@jeissonmoncada001 
Eslint not work

**Review by Julian9411 — APPROVED** [review]

**Pull request merged**